sessions/session-c.md

```diff
@@ -0,0 +1,48 @@
+# Session C: Meshing with cjdns and messing with secure-scuttlebutt
+
+## Description
+
+This hands-on session begins with [configuring a Raspberry Pi](https://github.com/tomeshnet/prototype-cjdns-pi2) as a [cjdns](https://github.com/cjdelisle/cjdns) mesh node. Then we will enable the IP tunnel feature to route all Internet traffic to another cjdns node through the [Hyperboria](https://github.com/hyperboria) encrypted mesh network. If you have a Raspberry Pi 3, please bring that along with [Raspbian Jessie Lite](https://www.raspberrypi.org/downloads/raspbian/) pre-flashed, but that is not a prerequisite to participate and there will be limited number of Raspberry Pi's provided at the workshop.
+
+We will run [secure-scuttlebutt](https://github.com/ssbc/secure-scuttlebutt) on our ad-hoc mesh, which keeps an append-only database on each node with peer-to-peer replication across the network, backing mesh-friendly applications such as [Patchwork](https://github.com/ssbc/patchwork) and [dnssb](https://github.com/ansuz/dnssb). Join us at this workshop for some DIY peer-to-peer social networking and DNS distribution!
```

Comment:
unclear what 'backing' means here? Also could we link out to wiki or explain what DNS is (or expand the acronym and link out)?